frontend/src/audios.html

```diff
@@ -20,13 +20,13 @@
   >
     <div class="flex-none bg-submarine-500 p-2 text-center font-bold uppercase text-neutral-600 md:hidden">Filter</div>
     <div class="grow overflow-y-auto overflow-x-hidden p-2 pb-40 pt-0">
-      <div id="language-list"></div>
-      <div id="sound-quality-list"></div>
-      <div id="duration-menu"></div>
-      <div id="percentage-menu"></div>
-      <div id="category-list"></div>
-      <div id="year-list"></div>
-      <div id="location-list"></div>
+      <div id="language-list">{% render 'partials/filter-section.html', title: 'Languages' -%}</div>
+      <div id="sound-quality-list">{% render 'partials/filter-section.html', title: 'Sound Quality' -%}</div>
+      <div id="duration-menu">{% render 'partials/filter-section.html', title: 'Duration' -%}</div>
+      <div id="percentage-menu">{% render 'partials/filter-section.html', title: 'Srila Gurudeva Timing' -%}</div>
+      <div id="category-list">{% render 'partials/filter-section.html', title: 'Category' -%}</div>
+      <div id="year-list">{% render 'partials/filter-section.html', title: 'Year' -%}</div>
+      <div id="location-list">{% render 'partials/filter-section.html', title: 'Location' -%}</div>
```

Comment:
Perhaps it will be easier to grasp this markup if we put the `render` tag on its own line.

Reply:
updated.